ex1/functions.py

```diff
@@ -65,24 +73,61 @@ def f():
 # compare against. The function should return a list with equal length as the
 # input list containing for each number in the original list either True or
 # False if the number was greater than or equal to n.
-
+def check_list(liste, n):
+    rlist= list()
+    for i in liste:
+        rlist.append(i>=n)
+    return rlist
+    
 # Define a function named check_list_nth that does the same as check_list but
 # uses every nth element of the input list (including the first one). You will
 # need a third input argument.
+def check_list_nth(liste, n, index):
```

Comment:
Please check against [the solution](https://github.com/TUW-GEO-python-intro/Exercise-1/pull/20/files#diff-7ec7b04e6cd2fbdbc70b7b64bf731ae8R188) since your function does not work.